modules/cnv-about-the-overview-dashboard.adoc

```diff
@@ -17,9 +18,9 @@ Status include *ok*, *error*, *warning*, *in progress*, and *unknown*. Resources
 ** Number of nodes
 ** Number of Pods
 ** Persistent storage volume claims
-** Virtual machines (available if {CNVProductName} is installed)
+** Virtual machines (available if container-native virtualization is installed)
 ** Bare metal hosts in the cluster, listed according to their state (only available in *metal3* environment).
-* *Cluster Health* summarizes the current health of the cluster as a whole, including relevant alerts and descriptions. If {CNVProductName} is installed, the overall health of {CNVProductName} is diagnosed as well. If more than one subsystem is present, click *See All* to view the status of each subsystem.
+* *Cluster Health* summarizes the current health of the cluster as a whole, including relevant alerts and descriptions. If container-native virtualization is installed, the overall health of container-native virtualization is diagnosed as well. If more than one subsystem is present, click *See All* to view the status of each subsystem.
```

Comment:
@vikram-redhat Are you okay with this change, or do you have another suggestion on how we should handle this for effective re-use?